src/com/google/javascript/jscomp/RewriteJsonToModule.java

```diff
@@ -36,6 +36,8 @@
     implements CompilerPass {
   public static final DiagnosticType JSON_UNEXPECTED_TOKEN =
       DiagnosticType.error("JSC_JSON_UNEXPECTED_TOKEN", "Unexpected JSON token");
+  public static final String JSC_BROWSER_BLACKLISTED_MARKER = "$jscomp$browser$blacklisted";
```

Comment:
It's not easy for us to add a dependency from the deps package onto the jscomp package.  Is it possible to move the definition of JSC_BROWSER_BLACKLISTED_MARKER into the deps package?